### DISCUSSION TITLE
Compatibility Raspi OS 12 Bookworm

### DISCUSSION THREAD

**Discussion by heinzgruber** [dtcooper/raspotify#651] [Q&A]
Hi,
is Rapotify compatible with Raspberry Pi OS 12 Bookworm? Ive read that the developers changed some audio related things but im no Linux expert. Or should i use Version 11 Bullseye?

Is for a Raspberry Pi Zero 2 32 or 64 Bit better for Raspotify? 

Thanks

**Comment by Brandon-Wass** [comment]
Short answer: Yes it is compatible.

More detailed: You have to make sure to edit the config file to point the audio to the correct device. Ex: "hw:hdmi-a-1". You can see a list of audio devices on your system by passing "librespot -d ?" without the quotation marks into command line after raspotify is installed. Once the correct device is indicated in the config file, simply restart the raspotify service or reboot your system and you should be good to go.

**Reply by heinzgruber** [reply]
Thanks for your answer. Already did exactly what you said and it's working. 👌🏻
